conf/application.conf

```diff
@@ -28,6 +28,7 @@ play.http.errorHandler = "uk.gov.hmrc.helplinefrontend.config.ErrorHandler"
 microservice {
 
   services {
+    find-your-national-insurance-number-host = "http://localhost:9000"
```

Comment:
I believe this should be in the format of:
```
services {
  microservice-name {
    host: ...
    port: ...
  }
  ...
```
as I think it affects what appears in the Service Relationships in The Catalogue

Reply:
Already covered by Chris' requested changes